tests/repr_test.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+import pandas as pd
+import pytest
+
+import narwhals.stable.v1 as nw
+
+
+def test_repr() -> None:
+    duckdb = pytest.importorskip("duckdb")
+    df = pd.DataFrame({"a": [1, 2, 3], "b": ["fdaf", "fda", "cf"]})
+    result = nw.from_native(df).__repr__()
+    expected = (
+        "┌──────────────────┐\n"
+        "|Narwhals DataFrame|\n"
+        "|------------------|\n"
+        "|       a     b    |\n"
+        "|    0  1  fdaf    |\n"
+        "|    1  2   fda    |\n"
+        "|    2  3    cf    |\n"
+        "└──────────────────┘"
+    )
+    assert result == expected
+    result = nw.from_native(df).lazy().__repr__()
+    expected = (
+        "┌──────────────────┐\n"
+        "|Narwhals LazyFrame|\n"
+        "|------------------|\n"
+        "|       a     b    |\n"
+        "|    0  1  fdaf    |\n"
+        "|    1  2   fda    |\n"
+        "|    2  3    cf    |\n"
+        "└──────────────────┘"
+    )
```

Comment:
I admit I panicked for a moment to see values in a lazyframe, but it's pandas so all good 😁